<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5BBDTNE" failures="2" tests="2" name="LoginTest" time="7.34" errors="0" timestamp="2025-06-05T19:13:49 BST">
  <testcase classname="tests.LoginPageTest" name="testValidLogin" time="0.802">
    <failure type="java.lang.AssertionError" message="expected [nopCommerce demo store. Login] but found [Just a moment...]">
      <![CDATA[java.lang.AssertionError: expected [nopCommerce demo store. Login] but found [Just a moment...]
at tests.LoginPageTest.testValidLogin(LoginTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testValidLogin -->
  <testcase classname="tests.LoginPageTest" name="testWithInValidCredentials" time="0.811">
    <failure type="java.lang.AssertionError" message="expected [Just a momnet....] but found [Just a moment...]">
      <![CDATA[java.lang.AssertionError: expected [Just a momnet....] but found [Just a moment...]
at tests.LoginPageTest.testWithInValidCredentials(LoginTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testWithInValidCredentials -->
</testsuite> <!-- LoginTest -->
